src/coord/src/coord.rs

```diff
@@ -2121,15 +2129,19 @@ impl Coordinator {
             return Err(CoordError::Eval(EvalError::NullCharacterNotPermitted));
```

Comment:
Oh, I realize this is from an earlier PR, but this is not the right error to return! This is about null characters in strings, not `Datum::Null`. I'd just use `coord_bail!` to write something like "secret value cannot be null."